python/lsst/synpipe/matchFakeStars.py

```diff
@@ -83,29 +87,29 @@ def main():
 
     #(starIndex,starList) = getFakeSources(args.rootDir, {'visit':args.visit, 'ccd':args.ccd})
     (starIndex, fakeXY, matchX, matchY, starPsfMag, starPsfMerr) = getFakeSources(args.rootDir,
-                                                                                  {'visit':args.visit, 'ccd':args.ccd})
+                                                                                  {'visit': args.visit, 'ccd': args.ccd})
 
     nInject = len(fakeXY)
-    nMatch  = len(np.argwhere(starPsfMag))
-    print "# Number of Injected Stars : %d" % nInject
-    print "# Number of Matched  Stars : %d" % nMatch
-    print "# Visit = %d   CCD = %d" % (args.visit, args.ccd)
-    print "# FakeX  FakeY  PSFMag  PSFMagErr  Deblend "
+    nMatch = len(np.argwhere(starPsfMag))
+    print("# Number of Injected Stars : %d" % nInject)
+    print("# Number of Matched  Stars : %d" % nMatch)
+    print("# Visit = %d   CCD = %d" % (args.visit, args.ccd))
+    print("# FakeX  FakeY  PSFMag  PSFMagErr  Deblend ")
 
     for i in range(nInject):
-       #print starIndex[i][0], starList[i]['flux.psf']
-       if len(starIndex[i]) > 1:
-           deblend = "blended"
-       elif starPsfMag[i] > 0:
-           deblend = "isolate"
-       else:
-           deblend = "nomatch"
+        #print starIndex[i][0], starList[i]['flux.psf']
```

Comment:
Any reason to keep this line?

Reply:
Nope, it is just something left over that I didnt catch